docs/mllib-migration-guides.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: global
+title: MLlib Migration
+displayTitle: MLlib Old Migration Guides
```

Comment:
Title should be consistent with displayTitle. In title, we want to move the most useful information to the front because the browser tabs are small. In displayTitle, we can include a link to the main MLlib guide, so there is a way to get back. For example, in mllib-clustering.md:

~~~
title: Clustering - MLlib
displayTitle: <a href="mllib-guide.html">MLlib</a> - Clustering
~~~